bin/tessel-2.js

```diff
@@ -287,18 +287,27 @@ module.exports.closeSuccessfulCommand = function() {
 };
 
 // Allow options to be partially applied
-module.exports.closeFailedCommand = function(opts, err) {
-  if (!err) {
-    err = opts;
-    opts = {};
-  }
-  if (err instanceof Error) {
-    throw err;
+module.exports.closeFailedCommand = function(opts, error) {
+
+  if (opts) {
```

Comment:
There were tests in `test/unit/bin-tessel-2.js` that broke when I assumed that couldn't happen. I'm going to investigate.